state-chain/pallets/cf-cfe-interface/src/lib.rs

```diff
@@ -11,17 +11,16 @@ use cf_chains::{
 };
 use cf_primitives::{Ed25519PublicKey, Ipv6Addr, Port};
 use cf_traits::{CfeBroadcastRequest, CfeMultisigRequest, CfePeerRegistration, Chainflip};
-use frame_support::{
-	pallet_prelude::Hooks,
-	storage::types::{StorageValue, ValueQuery},
-	traits::StorageVersion,
-};
+use frame_support::traits::StorageVersion;
 use frame_system::pallet_prelude::*;
 pub use pallet::*;
 use sp_std::vec::Vec;
 
 pub const PALLET_VERSION: StorageVersion = StorageVersion::new(0);
 
+// If any breaking change is made the CFE events, increment this version.
```

Comment:
```suggestion
// If any change is made to the CFE events, increment this version.
```

All changes to the CFE events are breaking to the CFE - possibly include that in the comment too

Reply:
In theory, you could remove some events from the end of the enum, and it wouldn't be a breaking change.